docs/deploy_features.md

```diff
@@ -128,7 +135,13 @@ By default, the deployed Azure web app uses the `text-embedding-ada-002` embeddi
     azd env set AZURE_OPENAI_EMB_DEPLOYMENT_VERSION 1
     ```
 
-4. When prompted during `azd up`, make sure to select a region for the OpenAI resource group location that supports the text-embedding-3 models. There are [limited regions available](https://learn.microsoft.com/azure/ai-services/openai/concepts/models#embeddings-models).
+4. Set the deployment Sku name to "GlobalStandard" :
```

Comment:
I realized that we dont yet have embedding models for GlobalStandard, so you can remove this bullet point for now. Can bring it back once we have those models. Still nice to have support for it in the Bicep though.